library/public/camera.h

```diff
@@ -57,6 +57,7 @@ class F3D_EXPORT camera
    * Angles are in degrees.
    */
   virtual camera& dolly(double val) = 0;
+  virtual camera& pan(double right, double up) = 0;
```

Comment:
I'll add a forward parameter for completeness
```suggestion
  virtual camera& pan(double right, double up, double forward = 0) = 0;
```